extensions/amp-ad-network-fake-impl/0.1/data/fake_amp_ad_with_iframe_transport.html

```diff
@@ -1,20 +1,25 @@
 <!doctype html>
 <html amp4ads>
   <head>
+    <title>Ad with 3p Analytics and amp-ad-exit</title>
     <meta charset=utf-8>
     <script async src='https://cdn.ampproject.org/v0.js'></script>
-    <script async custom-element=amp-analytics src='https://cdn.ampproject.org/v0/amp-analytics-0.1.js'></script>
+    <script async custom-element='amp-analytics' src='https://cdn.ampproject.org/v0/amp-analytics-0.1.js'></script>
+    <script async custom-element='amp-ad-exit' src='https://cdn.ampproject.org/v0/amp-ad-exit-0.1.js'></script>
     <style amp4ads-boilerplate>body{visibility:hidden}</style>
     <meta name=viewport content=width=device-width,minimum-scale=1>
   </head>
   <body>
-    <a href="/landing.html">
-      <amp-img id=img src='https://upload.wikimedia.org/wikipedia/commons/6/6e/Golde33443.jpg' width=256 height=300></amp-img>
-      <p class=frob>
-        By Golden Trvs Gol twister (Own work) [CC BY-SA 4.0 (http://creativecommons.org/licenses/by-sa/4.0)], via Wikimedia Commons
-      </p>
-    </a>
-    <amp-analytics type="testVendor">
+    Before using this example ad, please add the following to
+    extensions/amp-analytics/0.1/vendors.js:
```

Comment:
This is kludgey, but the other options are as follows and I don't like any of them, either:
* Allow the vendor to not be in vendors.js if running locally. Could cause false positives when running unit tests locally, if we write a unit test to ensure that vendor is valid (...which we should do)
* Use a real vendor in this example
* Hard-code an exception for this vendor name
* Add a function in vendors.js called addFakeVendor() or something
* Essentially subclass vendors.js and use the subclass in the examples. Somewhat tempting but would require propagating a change fairly deep that's not useful in production.